store/src/java/com/zimbra/cs/imap/ImapServerListenerPool.java

```diff
@@ -58,10 +58,14 @@ public static ImapServerListenerPool getInstance() {
 
     public ImapServerListener get(ZMailbox zmbox) throws ServiceException {
         Account acct = Provisioning.getInstance().get(AccountBy.id, zmbox.getAccountId());
+        if(acct == null) {
+            ZimbraLog.imap.error("Cannot get ImapServerListener for %s. Account does not exist.", zmbox.getAccountId(), null);
```

Comment:
Do we need the null as the 2nd parameter?  If you just want to log an error event without the exception message, I think you can call error(String format, Object ... objects) method.